lib/oxidized/model/riverbed.rb

```diff
@@ -0,0 +1,131 @@
+class Riverbed < Oxidized::Model
+  using Refinements
+
+  # Define the prompt
+  PROMPT = /^.*\s*[\w-]+\s*[#>]\s*$/i
+  prompt PROMPT
+
+  # Define comment character
+  comment '! '
+
+  # Use procs to remove unwanted lines
+  procs do
+    # Remove "Last login" message
+    proc { |output| output.gsub!(/^Last login:.*\n/, '') }
+  end
+
+  # Remove sensitive information
+  cmd :secret do |cfg|
+    cfg.gsub! /^(\s*tacacs-server (.+ )?key) .+/, '\\1 <secret hidden>'
```

Comment:
prefer space to \s when possible, as \s also matches \n and \r

Reply:
Replaced \s with explicit spaces to match only space characters.